src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    public static final double PI = 3.14159265358979323846;
+    private int radius;
+
+    public Circle() {
+        super();
+        this.radius = new Random().nextInt(100);
```

Comment:
Avoid using magic numbers directly in your code. Instead, define a constant for the maximum random radius value, e.g., `public static final int MAX_RADIUS = 100;`, and use it here.